src/averageAges.js

```diff
@@ -35,6 +44,15 @@ function calculateMenAverageAge(people, century) {
  */
 function calculateWomenAverageAge(people, withChildren) {
   // write code here
+  const women = withChildren
+    ? people.filter(person => {
+      return people.find(child => child.mother === person.name) !== undefined;
+    })
+    : people.filter(person => person.sex === 'f');
+
+  return women.reduce((age, person) => {
+    return age + (person.died - person.born);
+  }, 0) / women.length;
```

Comment:
```suggestion
  return women.reduce((age, person) => (
    age + (person.died - person.born);
  ), 0) / women.length;
```
можно в подобных ситуациях использовать еще вот такую запись